pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -103,9 +103,8 @@ export default class SettingsValidator {
         name: this.checkEnum('containerd', 'moby', 'docker'),
       },
       virtualMachine: {
-        memoryInGB:   this.checkLima(this.checkNumber(1, Number.POSITIVE_INFINITY)),
-        numberCPUs:   this.checkLima(this.checkNumber(1, Number.POSITIVE_INFINITY)),
-        hostResolver: this.checkPlatform('win32', this.checkBoolean),
```

Comment:
Shouldn't this be removed from the spec?
https://github.com/rancher-sandbox/rancher-desktop/blob/ce827861bc2f441f881c3f5dba7facf36f02ffcb/pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts#L99